pkg/server/etcd.go

```diff
@@ -1,103 +0,0 @@
-package server
-
-import (
-	"context"
-	"os"
-	"path/filepath"
-	"time"
-
-	"github.com/k3s-io/k3s/pkg/etcd"
-	"github.com/sirupsen/logrus"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-// setETCDLabelsAndAnnotations will set the etcd role label if not exists also it
```

Comment:
The code here only ran on apiserver+etcd nodes, while etcd-only nodes had a separate controller that managed their annotation:
https://github.com/k3s-io/k3s/blob/7739c8b97ee453a82d2762f2c2172c64e4aaa825/pkg/etcd/metadata_controller.go#L50

The etcd role annotation is now handled consistently, by running the etcd metadata controller on all server nodes when managed etcd is in use.